app/main.js

```diff
@@ -5,12 +5,14 @@ const path = require("path")
 const url = require("url")
 
 const childProcess = require("child_process")
-const electron = require("electron")
+const { app, ipcMain, nativeTheme, dialog, Menu, BrowserWindow, session } = require("electron")
```

Comment:
Please revert this change. I "required" all the modules this way intentionally to keep all the content of the modules in their own namespace respectively.

Reasons:

1) To not have to much identifiers in the global namespace
2) To know at first glance where functions, objects etc. belong to

Reply:
I have done so now, ( it fixed a bug originally too, to do with nativeTheme originally but when I reverted it somehow fixed the original issue )

Reply:
I tried running it on my laptop, and didn't throw any errors, ( never installed it there originally )